db/migrate/20210303124404_create_github_pull_requests.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGithubPullRequests < ActiveRecord::Migration[6.1]
+  def change
+    create_table :github_pull_requests do |t|
+      t.integer :github_id, null: false, index: { unique: true }
```

Comment:
Check these aren't bigint - I suspect they are.

Reply:
I didn't add the todo, but I was planning on using the node_id: https://docs.github.com/en/graphql/guides/using-global-node-ids I'm changing to code to work with that.

Reply:
Good catch. I've since changed the PR to use the node_id of a PR, which is a uniquely identifying string. It thus serves a similar function as the REST ID, but the latter can't be used in the graphql API whereas the former can. The REST API also returns the node_id, so I think this is probably the best identifier to use.